Source/CSBackend/Rendering/OpenGL/Base/GLContextRestorer.cpp

```diff
@@ -175,7 +175,7 @@ namespace CSBackend
         }
         
         //------------------------------------------------------------------------------
-        void GLContextRestorer::OnRenderSnapshot(ChilliSource::RenderSnapshot& renderSnapshot) noexcept
+        void GLContextRestorer::OnRenderSnapshot(ChilliSource::RenderSnapshot& renderSnapshot, ChilliSource::IAllocator* frameAllocator) noexcept
```

Comment:
How come allocator is now passed? render snapshot already contains a pointer to the relevant allocator.